README.rst

```diff
@@ -943,6 +944,46 @@ To use point ``STATICFILES_STORAGE`` to ``openwisp_utils.storage.CompressStaticF
 
     STATICFILES_STORAGE = 'openwisp_utils.storage.CompressStaticFilesStorage'
 
+Admin Theme utilities
+---------------------
+
+``openwisp_utils.admin_theme.email.send_email``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This function let you send customised mail using the template and logo that can be
+customised using `OPENWISP_EMAIL_TEMPLATE <#openwisp_email_template>`_ and
+`OPENWISP_EMAIL_LOGO <#openwisp_email_logo>`_ respectively.
+
+You can use `OPENWISP_HTML_EMAIL <#openwisp_html_email>`_ option to choose whether
+html template should be used or plain text should be sent.
+
+
+**Syntax:**
+
+.. code-block:: python
+
+    send_email(subject, body_text, body_html, recipients)
+
++--------------------+------------------------------------------------------------------------+
+| **Parameter**      | **Description**                                                        |
++--------------------+------------------------------------------------------------------------+
+| ``subject``        | (``str``) The subject of the email template.                           |
++--------------------+------------------------------------------------------------------------+
+| ``body_text``           | (``str``) The body of the text message to be emailed.             |
++--------------------+------------------------------------------------------------------------+
+| ``body_html``           | (``str``) The body of the html template to be emailed.            |
++--------------------+------------------------------------------------------------------------+
```

Comment:
please look at the resulting HTML https://github.com/openwisp/openwisp-utils/tree/issues/259-email-template#openwisp_utilsadmin_themeemailsend_email
It's not displaying properly.